hedera-node/hedera-app-spi/build.gradle.kts

```diff
@@ -20,5 +20,7 @@ plugins {
 description = "Hedera Application - SPI"
 
 dependencies {
+    api(libs.bundles.swirlds)
```

Comment:
I wonder if there is a more targeted lib we can use that only includes the merkle tree and virtual merkle tree APIs, so we're not pulling in all of swirlds

Reply:
I agree. `StateBuilder` feels out-of-place the way it is defined right now. We need to revisit this, when we tackle the state-functionality.

Reply:
BTW, I'm on a mission to terminate all third party dependencies I can. Maybe next year ;-)